test/powershell/Modules/Microsoft.Powershell.Host/Start-Transcript.Tests.ps1

```diff
@@ -136,4 +135,19 @@ Describe "Start-Transcript, Stop-Transcript tests" -tags "CI" {
         $transcriptFilePath | Should contain "PowerShell transcript end"
     }
 
+    It "Transcription should record native command output" {
+        try {
+            $ps = [powershell]::Create()
+            $ps.addscript("Start-Transcript -path $transcriptFilePath").Invoke()
+            $ps.addscript("hostname").Invoke()
+            $ps.addscript("Stop-Transcript").Invoke()
+        } finally {
+            if ($null -ne $ps) {
+                $ps.Dispose()
+            }
+        }
+        Test-Path $transcriptFilePath | Should be $true
```

Comment:
Also need to check that PS error stream is empty.

Reply:
will add

Reply:
Actually, there's no reason to follow that pattern, I'm rewriting it as a scriptblock and any errors should just show up in the test run.